tokio-trace/src/lib.rs

```diff
@@ -144,28 +143,17 @@ macro_rules! event {
     ($lvl:expr, { $($k:ident = $val:expr),* }, $($arg:tt)+ ) => (event!(target: None, $lvl, { $($k = $val),* }, $($arg)+))
 }
 
-lazy_static! {
```

Comment:
Nice removal!

Reply:
the implicit root span (and thus this static) are still there in this branch, just moved to the `span` module. as per our conversation on #4 I'll look into getting rid of it, but that'll happen in a separate PR...

Reply:
OH yeah! I saw later—my bad.